kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -537,8 +538,9 @@ private void enableInterface(String deviceId, NetworkProperties properties, Devi
     
     private void createVirtualInterface(String deviceId, NetworkProperties properties, NMDeviceType deviceType)
         throws DBusException {
+        NMVersion nmVersion = networkManager.getVersion();
         Map<String, Map<String, Variant<?>>> newConnectionSettings = NMSettingsConverter.buildSettings(properties,
-                Optional.empty(), deviceId, deviceId, deviceType);
+                Optional.empty(), deviceId, deviceId, deviceType, nmVersion);
```

Comment:
I would avoid creating a variable just for this (and think that Sonar would complain) therefore just pass `this.networkManager.getVersion()` to the `buildSettings` method